index.html

```diff
@@ -7691,7 +7691,7 @@ <h2>Acknowledgements</h2>
 Department of Homeland Security's Science and Technology Directorate under
 contracts HSHQDC-17-C-00019, 70RSAT20T00000010/P00001, 70RSAT20T00000029,
 70RSAT21T00000016/P00001, 70RSAT23T00000005, 70RSAT23C00000030,
-70RSAT23R00000006, and the National Science Foundation under NSF 22-572. The
+70RSAT23R00000006, 70RSAT22T00000001, and the National Science Foundation under NSF 22-572. The
 content of this specification does not necessarily reflect the position or the
 policy of the U.S. Government and no official endorsement should be inferred.
```

Comment:
Just manual linting.
```suggestion
70RSAT23R00000006, 70RSAT22T00000001, and the National Science Foundation
under NSF 22-572. The content of this specification does not necessarily
reflect the position or the policy of the U.S. Government and no official
endorsement should be inferred.
```

Reply:
I'll do this after I get the other PRs on contract acknowledgements in.